documentation/CAMARA-Security-Interoperability.md

```diff
@@ -195,15 +195,18 @@ Therefore scopes should be available to API implementations.
 
 ## Missing "openid" scope
 
-[OIDC Core Authentication Request](https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest) states the following about the value of scope.
+[OIDC Connect Authentication Request](https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest) states the following about the value of scope.
 > REQUIRED. OpenID Connect requests MUST contain the openid scope value. If the openid scope value is not present, the behavior is entirely unspecified. Other scope values MAY be present. ...
 
 This document defines that the authentication server SHOULD not return an id token if `openid` is missing in the scope parameter.
 
 This document defines the following error handling for a missing "openid" value in scope. 
-Please refer to [Authentication Error Response](https://openid.net/specs/openid-connect-core-1_0.html#AuthError).
 
-If "openid" is missing in the scope value but a claim that is [standardized in OIDC](https://openid.net/specs/openid-connect-core-1_0.html#Claims) is requested, then the Authorization Server returns an HTTP response code of 400 (Bad Request) and an error invalid_request.
+For OIDC please refer to [OIDC Authentication Error Response](https://openid.net/specs/openid-connect-core-1_0.html#AuthError).
```

Comment:
```suggestion
For the Authorization Code Flow please refer to [OIDC Authentication Error Response](https://openid.net/specs/openid-connect-core-1_0.html#AuthError).
```
We can say  Authorization Code Flow instead of OIDC. CIBA is also OpenId Connect. And the link points to section `3.1.2.6.  Authentication Error Response` of the OIDC Core standard which is under section `3.1.  Authentication using the Authorization Code Flow`.

Reply:
I think CIBA is not OIDC (OpenId Connect). They are separate standards published by the OIDF.

Reply:
Yes, of course, it is indeed a separate standard. What I mean is that it is called "**OpenID Connect** Client-Initiated Backchannel Authentication Flow - Core 1.0", so it looks better (or at least helpful) to me to explicitly mention the Authorization Code Flow instead of OIDC in this case, to avoid misunderstandings.

Reply:
But in any case, I don't think it's a big deal, so I've given my approval.